packages/vuetify/test/unit/components/VTimePicker/VTimePickerTitle.spec.js

```diff
@@ -2,126 +2,147 @@ import VTimePickerTitle from '@/components/VTimePicker/VTimePickerTitle'
 import { test } from '@/test'
 
 test('VTimePickerTitle.js', ({ mount }) => {
-  it('should render component in 24hr', () => {
-    const wrapper = mount(VTimePickerTitle, {
-      propsData: {
-        hour: 14,
-        minute: 13,
-        period: 'pm',
-        ampm: false
-      }
+  [true, false].forEach(useSecondsValue => {
```

Comment:
You can use [`describe.each()`](https://jestjs.io/docs/en/api#describeeachtable-name-fn-timeout) here

Reply:
I should be sure to rewrite it or let so will remain?

Reply:
I've never worked with unit tests before. 
TypeError: describe.each is not a function